lib/optimize/ConcatenatedModule.js

```diff
@@ -579,15 +600,19 @@ class ConcatenatedModule extends Module {
 					}
 				case "external":
 					{
-						info.interop = info.module.buildMeta && !info.module.buildMeta.exportsType;
 						const externalName = this.findNewName("", allUsedNames, null, info.module.readableIdentifier(requestShortener));
 						allUsedNames.add(externalName);
 						info.name = externalName;
-						if(info.interop) {
-							const externalNameInterop = this.findNewName("default", allUsedNames, null, info.module.readableIdentifier(requestShortener));
+						if(info.module.buildMeta.exportsType === "named" || !info.module.buildMeta.exportsType) {
```

Comment:
this looks a bit odd both if statements (this and next) will work if `!info.module.buildMeta.exportsType` ?

Reply:
Yep, that's intended. CJS (sometimes) need both interops. The first for `import * as X from "cjs"`, the second for `import X from "cjs"`.

Reply:
thanks, got it